qa/automation/CareerPortal.Tests.Scenarios/Scenarious/All_Buttons.feature

```diff
@@ -0,0 +1,44 @@
+Background:
+Given I open the https://career.quantori.com/ portal in Google Chrome
+
+Scenario 1.0: Cookie Test
+When I click on "Accept and Continue" button in pop-up cookie window 
```

Comment:
I agree with the reviewer, you should mention in Given section that cookies were not accepted before or you can add a step that user needs to clear cookies in his/her browser